apps/generator/test/test-project/test-project.test.js

```diff
@@ -15,10 +15,17 @@ const logMessage = require('../../lib/logMessages.js');
 const newContentNotExpectedInTest = 'new content';
 const version = '0.0.1';
 
-console.log = jest.fn();
+// Save the original console.log
+const originalConsoleLog = console.log;
+
+// Replace console.log with a custom mock function
+console.log = jest.fn((...args) => {
+  // Call the original function to actually log to the console
+  originalConsoleLog(...args);
+});
```

Comment:
sorry, I overlooked it, can you please put this logger hack also in `test-registry.test.js` file as tests fail there as well